xbmc/GUIInfoManager.cpp

```diff
@@ -6862,215 +6870,217 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
-                                  { "addonsummary",     LISTITEM_ADDON_SUMMARY },
-                                  { "addondescription", LISTITEM_ADDON_DESCRIPTION },
-                                  { "addondisclaimer",  LISTITEM_ADDON_DISCLAIMER },
-                                  { "addonnews",        LISTITEM_ADDON_NEWS },
-                                  { "addonbroken",      LISTITEM_ADDON_BROKEN },
-                                  { "addonlifecycletype", LISTITEM_ADDON_LIFECYCLE_TYPE },
-                                  { "addonlifecycledesc", LISTITEM_ADDON_LIFECYCLE_DESC },
-                                  { "addontype",        LISTITEM_ADDON_TYPE },
-                                  { "addoninstalldate", LISTITEM_ADDON_INSTALL_DATE },
-                                  { "addonlastupdated", LISTITEM_ADDON_LAST_UPDATED },
-                                  { "addonlastused",    LISTITEM_ADDON_LAST_USED },
-                                  { "addonorigin",      LISTITEM_ADDON_ORIGIN },
-                                  { "addonsize",        LISTITEM_ADDON_SIZE },
-                                  { "expirationdate",   LISTITEM_EXPIRATION_DATE },
-                                  { "expirationtime",   LISTITEM_EXPIRATION_TIME },
-                                  { "art",              LISTITEM_ART },
-                                  { "property",         LISTITEM_PROPERTY },
-                                  { "parentalrating",   LISTITEM_PARENTAL_RATING },
-                                  { "currentitem",      LISTITEM_CURRENTITEM },
-                                  { "isnew",            LISTITEM_IS_NEW },
-                                  { "isboxset",         LISTITEM_IS_BOXSET },
-                                  { "totaldiscs",       LISTITEM_TOTALDISCS },
-                                  { "releasedate",      LISTITEM_RELEASEDATE },
-                                  { "originaldate",     LISTITEM_ORIGINALDATE },
-                                  { "bpm",              LISTITEM_BPM },
-                                  { "uniqueid",         LISTITEM_UNIQUEID },
-                                  { "bitrate",          LISTITEM_BITRATE },
-                                  { "samplerate",       LISTITEM_SAMPLERATE },
-                                  { "musicchannels",    LISTITEM_MUSICCHANNELS },
-                                  { "ispremiere",       LISTITEM_IS_PREMIERE },
-                                  { "isfinale",         LISTITEM_IS_FINALE },
-                                  { "islive",           LISTITEM_IS_LIVE },
-                                  { "tvshowdbid",       LISTITEM_TVSHOWDBID },
-                                  { "albumstatus",      LISTITEM_ALBUMSTATUS },
-                                  { "isautoupdateable", LISTITEM_ISAUTOUPDATEABLE },
-                                  { "hdrtype",          LISTITEM_VIDEO_HDR_TYPE },
+const infomap listitem_labels[] = {
```

Comment:
Please add `// clang-off` at the beginning and `// clang on` at the end to avoid the giant diff

Reply:
Better would be to have a first commit that applies formatting, and a second commit that gives you a nice beautiful green addition without any surrounding red.